catalyst_voices/packages/internal/catalyst_voices_repositories/lib/src/dto/document_schema_dto.dart

```diff
@@ -0,0 +1,388 @@
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_repositories/src/dto/document_definitions_dto.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'document_schema_dto.g.dart';
+
+@JsonSerializable()
+class DocumentSchemaDto extends Equatable {
+  @JsonKey(name: r'$schema')
+  final String schema;
+  @JsonKey(name: r'$id')
+  final String id;
+  final String title;
+  final String description;
+  final DocumentDefinitionsDto definitions;
+  final String type;
+  final bool additionalProperties;
+  @JsonKey(
+    toJson: _toJsonProperties,
+    fromJson: _fromJsonProperties,
+    name: 'properties',
+  )
+  final List<DocumentSchemaSegmentDto> segments;
+  @JsonKey(name: 'x-order')
+  final List<String> order;
+  @JsonKey(includeToJson: false)
+  final String propertiesSchema;
+
+  static late Map<String, int> orderMap;
```

Comment:
Please make it just a field in this class, non-static one. Make it immutable as well.

Making it mutable static exposes too much and allows you to modify/access it from anywhere